src/Storage/Storage.Management/help/New-AzDataLakeGen2SasToken.md

```diff
@@ -0,0 +1,238 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/powershell/module/az.storage/new-azdatalakegen2sastoken
+schema: 2.0.0
+---
+
+# New-AzDataLakeGen2SasToken
+
+## SYNOPSIS
+Generates a SAS token for Azure DatalakeGen2 item.
+
+## SYNTAX
+
+### ReceiveManual (Default)
+```
+New-AzDataLakeGen2SasToken [-FileSystem] <String> [-Path <String>] [-Permission <String>]
+ [-Protocol <SasProtocol>] [-IPAddressOrRange <String>] [-StartTime <DateTimeOffset>]
+ [-ExpiryTime <DateTimeOffset>] [-FullUri] [-Context <IStorageContext>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ItemPipeline
+```
+New-AzDataLakeGen2SasToken -InputObject <AzureDataLakeGen2Item> [-Permission <String>]
+ [-Protocol <SasProtocol>] [-IPAddressOrRange <String>] [-StartTime <DateTimeOffset>]
+ [-ExpiryTime <DateTimeOffset>] [-FullUri] [-Context <IStorageContext>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzDataLakeGen2SasToken** cmdlet generates a Shared Access Signature (SAS) token for an Azure DatalakeGen2 item.
+
+## EXAMPLES
+
+### Example 1: Generate a SAS token with full permission
+```
+PS C:\> New-AzDataLakeGen2SasToken -FileSystem "filesystem1" -Path "dir1/dir2" -Permission racwdlmeop
```

Comment:
Please remove the prefix: `PS C:\>` here.

Reply:
fixed